### PR TITLE
mgr/dashboard_v2: Auto refresh hosts

### PR DESCRIPTION
**Pull request by ricardoasmarques**
Hosts list should auto refresh every 5 seconds (same behavior as legacy dashboard).

Signed-off-by: Ricardo Marques <rimarques@suse.com>

### CODE REVIEW

**Review by LenzGr — DISMISSED** [review]

File: src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts [hunk 1]
```diff
@@ -39,6 +40,13 @@ export class HostsComponent implements OnInit {
         pipe: this.cephShortVersionPipe
       }
     ];
+    this.interval = setInterval(() => {
```
Comment (votdev) [inline src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts:44]: Does it make sense to add a mechanism which makes sure that the data is not fetched during another fetch is in progress?

**Review by votdev — CHANGES_REQUESTED** [review]

File: src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts (hunk 1 above)
Comment (ricardoasmarques) [reply src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts:44]: @votdev I agree, but we have the same problem on all other pages. Shouldn't we merge this PR as is, and I will fix the "stacked calls" problem on a separate PR for every page?

Comment (ricardoasmarques) [reply src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts:44]: Ignore my previous comment, I will fix the hosts page on this PR, and others will be fixed on a separate PR.

Comment (votdev) [reply src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts:44]: OK, i'm fine with both proposals.

Comment (votdev) [reply src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts:44]: We can use your improvements for this PR as a template for the other pages then.

**Review by votdev — DISMISSED** [review]

File: src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts (hunk 1 above)
Comment (ricardoasmarques) [reply src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/cluster/hosts/hosts.component.ts:44]: @votdev Now I'm preventing multiple executions of the fetch data in parallel. 

I've also added a commit that prevent double click on datatable refresh button.

**Review by votdev — APPROVED** [review]

**Pull request merged**